extensions/amp-analytics/0.1/vendors.js

```diff
@@ -46,17 +47,20 @@ export const ANALYTICS_CONFIG = {
   },
 
   'googleanalytics': {
-    'host': 'www.google-analytics.com',
-    'method': 'GET',
     'requests': {
-      'baseHit': '/collect?v=1&_v=a0&aip=true&_s=${hitCount}&' +
-          'dl=${canonicalUrl}&dt=${title}&sr=${screenWidth}x${screenHeight}&' +
-          '_utmht=${timestamp}&jid=&cid=${clientId(_ga)}&tid=${account}',
-      'pageview': '/r${baseHit}&t=pageview&_r=1',
-      'event': '${baseHit}&t=event&ec=${eventCategory}&ea=${eventAction}&' +
-          'el=${eventLabel}&ev=${eventValue}',
-      'social': '${baseHit}&t=social&sa=${socialAction}&sn=${socialNetwork}&' +
-          'st=${socialActionTarget}'
+      'commonHost': 'https://www.google-analytics.com',
```

Comment:
Those are request parts common to all the other requests. (The expansion scope of these are just the requests object itself.)